docs/spec/instrument/hooks.md

```diff
@@ -1,6 +1,733 @@
 # Supported hooks
```

Comment:
Could you look at this table here please and try to merge the two perspectives?
https://github.com/zenitysec/AOS/blob/dev/docs/topics/core_concepts.md#agent-environment-overview

Reply:
wdyt about the current in general?
still need to specify detailed mcp hooks.

Reply:
not sure I understand the question